requirements.txt

```diff
@@ -1,9 +1,10 @@
-fastdtw==0.3.4
-mediapipe==0.8.7.3
+fastdtw
+mediapipe
 numpy
-opencv-contrib-python==4.5.4.60
-opencv-python==4.5.3.56
-pandas==1.3.2
-pytube==11.0.1
-tdqm==0.0.1
-flask
\ No newline at end of file
+opencv-contrib-python
+opencv-python
+pandas
+pytube
+tdqm
+flask
+yt-dlp
```

Comment:
bs4, requests are required but not in requirements.txt

Reply:
Added in new commits